counter/package.json

```diff
@@ -1,29 +1,24 @@
 {
     "scripts": {
-        "start": "npx webpack-dev-server",
-        "build": "npx webpack"
+        "postinstall": "bower install bower.json && bower install",
+        "start": "pulp build && concurrently \"pulp --watch build\" \"parcel index.html\""
     },
     "devDependencies": {
         "@babel/core": "^7.4.4",
-        "@babel/preset-env": "^7.4.4",
         "@babel/preset-react": "^7.0.0",
-        "babel-loader": "^8.0.5",
+        "bower": "^1.8.8",
+        "concurrently": "^4.1.0",
+        "parcel": "^1.12.3",
         "pulp": "^12.4.2",
         "purescript": "0.12.5",
         "purescript-psa": "0.7.3",
-        "purs-loader": "^3.4.0",
-        "request": "^2.88.0",
-        "webpack": "^4.30.0",
-        "webpack-cli": "^3.3.2",
-        "webpack-dev-server": "^3.3.1",
-        "yarn": "1.9.4"
+        "request": "^2.88.0"
```

Comment:
What is `request` used for?

Reply:
I'm not quite sure, but purescript refused to install without it. The installation script apparently needs `request` for something